benefits/core/migrations/0002_sample_data.py

```diff
@@ -0,0 +1,224 @@
+from django.db import migrations
```

Comment:
This is awesome! Just to organize things a bit (and allow us to test if we wanted) can you please split each model creation (`PemData`, `EligibilityType`, etc.) into its own function and call each from `load_initial_data`?

Reply:
Or alternatively, use multiple `migrations.RunPython` steps.

Reply:
I thought about this too but held off on it because some models need references to ones created previously. I can certainly return those models after they're created and then pass them to the later models, but there are some assumptions made about ordering that I feel are a bit brittle. Maybe I can push an attempt and get your feedback on it.

Reply:
Ah, interesting - that's right. 

I actually assumed we would have multiple _migration files_, and each would list the dependencies it has. But this is probably closer to how we'd do it in production (a single file with all data) so this is fine 👍

Reply:
Do you think it's ok to leave it as-is? Or should I try to split up into functions?

Reply:
Naw it's all good!